cli/templates/containers/pageViewTemplate.ts

```diff
@@ -0,0 +1,21 @@
+const pageViewTemplate = (pageName) => {
+  return `import React from 'react'
+
+import { I${pageName}PageProps } from '../../pages'
```

Comment:
`import { I${pageName}PageProps } from '../../pages'` not all of the PageProps will be in the index file. `pages/${pageName}` is missing.